ovos_plugin_manager/templates/tts.py

```diff
@@ -359,6 +168,26 @@ def get_tts(self, sentence, wav_file, lang=None):
         """
         return "", None
 
+    def get_voice(self, gender, lang=None):
+        """ map a language and gender to a valid voice for this TTS engine """
+        lang = lang or self.lang
+        return gender
```

Comment:
Not clear on the purpose of this, you only return gender even though you make sure there's a value for lang

Reply:
this is meant for plugins to override, given a preference of voice gender the plugin should return a default voice with that gender

we dont really use this anywhere, i think it was introduced for Neon some time back? i wouldnt mind deleting it, but its already there so...

Reply:
I'm curious to see how Neon is implementing it and maybe making this overridable version more like theirs. Right now it doesn't make any sense to accept a language, assign it a value, and then throw it away...